content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md

```diff
@@ -25,9 +25,11 @@ sudo apt-get update; \
 dotnet --list-sdks
 ```
 - It should return `5.0.202 [/usr/share/dotnet/sdk]` or a newer version of .NET 5.
-- Then, use the following command to install a new project template.
+
+# Install Orchard Template
+- Then, use the following command to install a new project template (In this case, it'll use [nightly build](https://github.com/OrchardCMS/OrchardCore/#build-status) for OrchardCore).
 ```sh
-dotnet new -i OrchardCore.ProjectTemplates::1.0.0-rc2-* --nuget-source https://nuget.cloudsmith.io/orchardcore/preview/v3/index.json
+dotnet new -i "OrchardCore.ProjectTemplates::1.0.0-rc2-*" --nuget-source https://nuget.cloudsmith.io/orchardcore/preview/v3/index.json
```

Comment:
Do we really need quotes for a  template name? I think having quotes is safe but just curiosity, if without it still work.

Reply:
Without quote, it'll get error on WSL2 ;)

Reply:
It got error, because I use `zsh` Lol

Reply:
I think with a quote, it's safer.

Reply:
Amazing, I think it use it before and don't have any issue.
Anyway, it is safer to have quote. Thanks.